src/ast/nodes/ExpressionStatement.ts

```diff
@@ -1,14 +1,24 @@
 import type MagicString from 'magic-string';
 import { errorModuleLevelDirective } from '../../utils/error';
 import type { RenderOptions } from '../../utils/renderHelpers';
+import type { DeoptimizableEntity } from '../DeoptimizableEntity';
 import type { InclusionContext } from '../ExecutionContext';
+import type { ObjectPath, PathTracker } from '../utils/PathTracker';
 import * as NodeType from './NodeType';
 import { type ExpressionNode, StatementBase } from './shared/Node';
 
 export default class ExpressionStatement extends StatementBase {
 	declare directive?: string;
 	declare expression: ExpressionNode;
 
+	getLiteralValueAtPath(
+		path: ObjectPath,
+		recursionTracker: PathTracker,
+		origin: DeoptimizableEntity
+	) {
+		return this.expression.getLiteralValueAtPath(path, recursionTracker, origin);
```

Comment:
Why did you add this one? I cannot think of a scenario where the literal value of an ExpressionStatement would be queried as it cannot be nested in other expressions except `BlockStatement` and that one does not care about values. Maybe we can remove it.

Reply:
Removed. I wasn't familiar the pipeline so I tried different places :P